addOns/automation/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Added
+- Updated automation framework documentation and YAML files for `activeScan` job to reflect PR #4326
```

Comment:
This should be in Changed or even Fixed not Added, the PR reference is unnecessary. "YAML files", should be "templates"